### PR TITLE
Remove DO_NOT_TEST from config unit tests

### PR DESCRIPTION
**Pull request by jdavcs**
100% of schema-derived config properties are covered by unit tests. 
Detailed comments in commit messages.

### CODE REVIEW

File: doc/source/admin/galaxy_options.rst [hunk 1]
```diff
@@ -1381,9 +1381,7 @@
 ~~~~~~~~~~~~~~~~
 
 :Description:
-    This flag enables an AWS cost estimate for every job based on their runtime matrices.
-    CPU, RAM and runtime usage is mapped against AWS pricing table.
-    Please note, that those numbers are only estimates.
+    Enable AWS estimate.
```
Comment (nsoranzo) [inline doc/source/admin/galaxy_options.rst]: I think the proper fix here would be to copy the long option description to `lib/galaxy/webapps/galaxy/config_schema.yml`

File: lib/galaxy/webapps/galaxy/config_schema.yml [hunk 2]
```diff
@@ -2841,14 +2839,13 @@ mapping:
...
         required: false
         desc: |
           This parameter describes how to map users and workflows to a set of workflow
           resource parameter to present (typically input IDs from workflow_resource_params_file).
           If this this is a function reference it will be passed various inputs (workflow model
           object and user) and it should produce a list of input IDs. If it is a path
-          it is expected to an XML or YAML file describing how to map group names to parameter
+          it is expected to be an XML or YAML file describing how to map group names to parameter
           descriptions (additional types of mappings via these files could be implemented but
           haven't yet - for instance using workflow tags to do the mapping).
```
Comment (nsoranzo) [inline lib/galaxy/webapps/galaxy/config_schema.yml:2852]: I think this help should mention the that `config/workflow_resource_mapper_conf.yml.sample` is an example of such a config file.

File: doc/source/admin/galaxy_options.rst (hunk 1 above)
Comment (jdavcs) [reply doc/source/admin/galaxy_options.rst]: Done. It appears that text was only in `galaxy_options.rst`, so it got overwritten on rebuild. I assumed that was an updated/shortened description. Thanks!

File: lib/galaxy/webapps/galaxy/config_schema.yml (hunk 2 above)
Comment (jdavcs) [reply lib/galaxy/webapps/galaxy/config_schema.yml:2852]: Done. Added as "sample default path" to the description.

File: lib/galaxy/webapps/galaxy/config_schema.yml [hunk 3]
```diff
@@ -2841,17 +2841,18 @@ mapping:
...
 
+          Sample default path 'config/workflow_resource_mapper_conf.yml'
```
Comment (nsoranzo) [inline lib/galaxy/webapps/galaxy/config_schema.yml]:
```suggestion
          Sample default path 'config/workflow_resource_mapper_conf.yml.sample'
```

**Comment by ic4f** [comment]
Thank you!

**Pull request merged**